package.json

```diff
@@ -137,7 +138,7 @@
     "@types/js-md5": "0.4.3",
     "@types/lodash": "4.14.182",
     "@types/punycode": "2.1.0",
-    "@types/react": "17.0.14",
+    "@types/react": "18.0.24",
```

Comment:
IS this necessary? We don't use any React 18 features and the update creates a lot of noise, last I checked.